File: CPP/Aksesoris.cpp
#include <bits/stdc++.h>
#include "PetShop.cpp"
using namespace std;

class Aksesoris: public PetShop {
  protected:
    string jenis;
    string warna;
    string bahan;
  public:
  Aksesoris(){
  }

  Aksesoris(int id, string name, string category, int price, string j, string w, string b) : PetShop(id , name, category, price){
    this->jenis = j;
    this->warna = w;
    this->bahan = b;
  }

  void setJenis(string j){
    this->jenis = j;
  }

  string getJenis(){
    return this->jenis;
  }

  void setWarna(string w){
    this->warna = w;
  }

  string getWarna(){
    return this->warna;
  }

  void setBahan(string b){
    this->bahan = b;
  }

  string getBahan(){
    return this->bahan;
  }

  ~Aksesoris(){
  }
};

File: PHP/Aksesoris.php
<?php
require_once("PetShop.php");
class Aksesoris extends PetShop {
  protected $jenis;
  protected $bahan;
  protected $warna;

  public function __construct($id, $nama_produk, $harga_produk, $stok_produk, $foto_produk, $jenis, $bahan, $warna) {
    parent::__construct($id, $nama_produk, $harga_produk, $stok_produk, $foto_produk);
    $this->jenis = $jenis;
    $this->bahan = $bahan;
    $this->warna = $warna;
  }

  public function getJenis() {
    return $this->jenis;
  }

  public function getBahan() {
    return $this->bahan;
  }

  public function getWarna() {
    return $this->warna;
  }

  public function setJenis($jenis) {
    $this->jenis = $jenis;
  }

  public function setBahan($bahan) {
    $this->bahan = $bahan;
  }

  public function setWarna($warna) {
    $this->warna = $warna;
  }

  public function print() {
    parent::print();
    echo "Jenis: " . $this->jenis . "<br>";
    echo "Bahan: " . $this->bahan . "<br>";
    echo "Warna: " . $this->warna . "<br>";
  }
}


?>